python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -19,7 +19,7 @@ def execute(config, project, name, entities, base_configuration, engine_name):
 
     # Setup the bootstrap manager.
     toolkit_mgr = sgtk.bootstrap.ToolkitManager()
-    toolkit_mgr.plugin_id = "shotgun_toolkit_command_execution"
+    toolkit_mgr.plugin_id = "basic.shotgun.execute"
```

Comment:
i think you are right @thebeeland - safer to have two processes like this - please add a comment to explain. nice one!